packages/vm/lib/runBlock.ts

```diff
@@ -253,11 +253,6 @@ async function applyTransactions(this: VM, block: Block, opts: RunBlockOpts) {
   for (let txIdx = 0; txIdx < block.transactions.length; txIdx++) {
     const tx = block.transactions[txIdx]
 
-    const gasLimitIsHigherThanBlock = block.header.gasLimit.lt(tx.gasLimit.add(gasUsed))
```

Comment:
Note that validating that the tx's gas limit isn't larger than the block's is done in runTx.